pages/DynamicPOC/index.tsx

```diff
@@ -55,50 +55,67 @@ const TitleText = styled.h1`
   font-size: large;
 `
 
-const DynamicPOC: React.FC = () => {
+const files = {
+  animalForm: '../../../constants/Animal Form.csv',
+  actualForm: '../../../constants/EdLaw Combined Flowchart.csv',
+}
+
```

Comment:
What exactly are props/what is this returning?

Reply:
this is from my PR so I can explain -
`props`: Properties that are passed into a page/component
`static props`: Properties that are the same for everyone (regardless of user)
`getStaticProps`: a method that allows us to pass static props into this page and run the bulk of the fetching on the server. You can read a little more [here](https://nextjs.org/docs/basic-features/data-fetching/get-static-props)

The reason getStaticProps is used here is that the csv parsing needs to occur on the server side due to some of the packages it uses. By running that logic in getStaticProps, it will run on the server side but still be provided to our component (`DynamicPOC`)